traffic_stops/static/js/app/common/IncidentByReasonAndRace.js

```diff
@@ -251,17 +251,17 @@ export const IRRTableBase = TableBase.extend({
     this.add_select();
   },
 
-  _reasons: _.memoize(function () {
+  _reasons: function () {
     return d3.set(_.pluck(this._raw_data(), this.reason_type));
-  }),
+  },
 
   add_select: function () {
     let div = $(this.get("selector"));
 
     // select input  only needs to be added once
     if (div.find('select').length) { return true; }
 
-    let $selector = $('<select id="srr-table-select">');
+    let $selector = $(`<select id="${this.get('selector').replace('#', '')}_select">`);
```

Comment:
Yeah, that's what this is supposed to do. Instead of just creating multiple elements with the id `srr-table-select`, we'll now have more specifically named elements.

Nothing actually *looks for* these IDs, so it's a bit of a mystery why we even bother—the other class in this same module actually doesn't. But it's a bit of dust that we can sweep up.